alerter/src/main/java/org/apache/hertzbeat/alert/AlerterWorkerPool.java

```diff
@@ -50,11 +50,11 @@ private void initWorkExecutor() {
                 .setDaemon(true)
                 .setNameFormat("alerter-worker-%d")
                 .build();
-        workerExecutor = new ThreadPoolExecutor(6,
+        workerExecutor = new ThreadPoolExecutor(10,
                 10,
                 10,
                 TimeUnit.SECONDS,
-                new SynchronousQueue<>(),
+                new LinkedBlockingQueue<>(),
```

Comment:
It is dangerous here, because the default size of `LinkedBlockingQueue` is `Integer.MAX_VALUE`.
It may cause OOM

Reply:
Hi, here the threads can be more than 10 at the same time, but the number is unknown(not too much), so linkedqueue is used.